### ISSUE TITLE
Phone field error messages not translatable

### ISSUE THREAD

**Issue by bavocorteville** [verbb/formie#696]
**Description**

The error messages from the phone field are not added to `window.FormieTranslations`:

- Invalid number, 
- Invalid country code, 
- Too short, 
- Too long

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.3",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.5.4](https://github.com/verbb/formie/releases/tag/1.5.4)
